node-bootstrapper/main.go

```diff
@@ -217,8 +279,28 @@ type File struct {
 	Mode    os.FileMode
 }
 
-func customData(config *datamodel.NodeBootstrappingConfiguration) (map[string]File, error) {
-	contentDockerDaemon, err := generateContentDockerDaemonJSON(config)
+func customData(ctx context.Context, config *datamodel.NodeBootstrappingConfiguration) (map[string]File, error) {
+	if config.AgentPoolProfile.IsWindows() {
+		customData, err2 := OldCustomData(ctx, config)
+		if err2 != nil {
+			log.Fatal("error:", err2)
```

Comment:
Avoid using log.Fatal unless absolutely necessary. It exists application immediately, without calling any `defer` calls.
We using different logger. 

Also, `err2`?